crates/ty_python_semantic/resources/primer/bad.txt

```diff
@@ -1,2 +1,2 @@
 spark  # too many iterations (in `exported_names` query), `should not be able to access instance member `spark` of type variable IndexOpsLike@astype in inferable position`
```

Comment:
The second of the two panics mentioned here is presumably gone after Doug's recent work in https://github.com/astral-sh/ruff/pull/20677?